pages/vi/find-pi.md

```diff
@@ -34,6 +34,7 @@ You can stop this process with `Ctrl`+`C`.
 As you can see here its local IP address is `192.168.0.19`.
 
 ## Method 3: nmap command
++ Note: If this is your first time using the CLI on macOS you need to install brew, a package management system for the CLI
```

Comment:
@rohansuri17 this is not an mdwiki note please to other md files how it is solved there

https://github.com/treehouses/treehouses.github.io/search?q=Note&unscoped_q=Note